pyproject.toml

```diff
@@ -13,8 +13,8 @@ packages = [
 terralab = "terralab.cli:cli"
 
 [tool.poetry.dependencies]
-python = "^3.9"
-terra-scientific-pipelines-service-api-client = "^0.1.3"
+python = "^3.12"
```

Comment:
with this change, we should also remove versions 3.9-3.11 from the list in `sonar-project.properties` (unclear to me what that was doing but at least let's keep it consistent 😬 )

Reply:
great point, removing